masonry/src/testing/harness.rs

```diff
@@ -336,7 +336,7 @@ impl TestHarness {
 
     // --- MARK: EVENT HELPERS ---
 
-    /// Move an internal mouse state, and send a MouseMove event to the window.
+    /// Move an internal mouse state, and send a [`PointerEvent::PointerMove`] event to the window.
```

Comment:
```suggestion
    /// Move an internal mouse state, and send a [`PointerMove`](PointerEvent::PointerMove) event to the window.
```
might be slightly nicer formatting.